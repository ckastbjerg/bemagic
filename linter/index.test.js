/* eslint-env jest */
const linter = require('./index');
const dummyBlock = require('../dummyBlock');

test('it works', () => {
    expect(linter([dummyBlock])).toMatchSnapshot();
});
