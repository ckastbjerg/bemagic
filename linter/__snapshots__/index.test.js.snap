exports[`test it works 1`] = `undefined`;
